### DISCUSSION TITLE
How to pin LazyVim to 12.x

### DISCUSSION THREAD

**Discussion by diocletiann** [LazyVim/LazyVim#4725] [Q&A] [answered]
I just updated to 13 and don't have time to fix the breakage, how do I pin to 12.x so I can roll back?

**Comment by dpetka2001** [comment, marked as answer]
13.x is not released yet. You must have something like [this](https://www.lazyvim.org/configuration/lazy.nvim) in your personal configuration and you updated to latest HEAD. Just comment out those lines wherever you have them in your personal configuration and restart Neovim and press `C` and `U` in Lazy UI to update to the current stable version. But tomorrow will probably be the new release 13.0, so just put `version = "12.44.1"`, which is the latest stable as of today.

**Reply by diocletiann** [reply]
thanks! I had this at the top of `plugins.lua`
```
return {
    {
        "LazyVim/LazyVim",
        version = false,
        ...
```
